### ISSUE TITLE
Error loading configuration file

### ISSUE THREAD

**Issue by jparisu** [markdownlint/markdownlint#456]
## Description

Using command `mdl --config <file> <files>`, the configuration file is supposed to be used, but is not.

### Environment

Ubuntu 22.04

**MDL Version**
`v0.12.0`

### Expected Behavior

Using a configuration file as follows:

```rb
# cfg.rb
# Add all rules
all

# Set minimum identation list as 2
rule 'MD007', :indent => 2

# Exclude line lenght check
exclude_rule 'MD013'

# Exclude ordered list item prefixt check
# Otherwise every markdown should use only one of the following styles, and bot are desired:
# - [1. 1. 1. ...]
# - [1. 2. 3. ...]
exclude_rule 'MD029'
```

I would expect that using this config file with this command, errors of `MD013` does not appear:

`mdl -c cfg.rb README.md --verbose`

### Actual Behavior

This is the output I am getting from such command:

```sh
Loaded config from <...>/cfg.rb
Checking README.md...
Processing rule MD001
...
Processing rule MD013
README.md:5: MD013 Line length
README.md:6: MD013 Line length
README.md:17: MD013 Line length
README.md:21: MD013 Line length
README.md:22: MD013 Line length
README.md:39: MD013 Line length
README.md:45: MD013 Line length
README.md:64: MD013 Line length
README.md:72: MD013 Line length
Processing rule MD014
...
```

## Replication Case

In action https://github.com/eProsima/eProsima-CI/actions/runs/4988717407/jobs/8931737963 you can see an example of this happeninng.

## Notes

Doing this in the script: `echo "style 'cfg.rb'" > .mdlrc` I handled to make it work.
So using the default configuration file `.mdlrc` correctly set the configuration and the rules.

**Comment by jaymzh** [comment]
That's a style, not a config. So this isn't a bug.

Put all that in to `style.rb`, and then in `cfg.rb` put `style 'style.rb'`, then you can do `-c cfg.rb`

**Comment by jparisu** [comment]
> That's a style, not a config. So this isn't a bug.

That is true, my fault.
However it would be fine to be able to configure the style file as another argument, as it is possible with configuration file.

Thanks for the help and for the project :smiley:
